OracleSOA/dockerfiles/12.2.1.2/Dockerfile.quickstart

```diff
@@ -0,0 +1,40 @@
+# Pull base image
+# ---------------
+FROM oracle/serverjre:8
```

Comment:
No, I mean take a look at the new WebLogic Dockerfiles: @mriccell did a great job to restructure them so that there is more layer re-use between versions. I think the same techniques could be used here.